llvm/docs/GettingInvolved.rst

```diff
@@ -331,6 +331,11 @@ don't find anyone present, chances are they happen to be off that day.
     - Monthly, 4th Wednesday of the month at 9:30am PT, for 30 minutes.
     - `Google meet <https://meet.google.com/pdd-dibg-cwv>`__
     - English
+  * - Rotating hosts
+    - Getting Started, beginner questions, new contributors.

```

Comment:
You might want to specify the scope a bit more, like are these focused on LLVM, on Clang, how to use GitHub, or sort of all-of-the-above-plus-more?

Reply:
I was actually hoping that intentionally vague could be a feature not a bug! For the newest contributors that might not know which of the above office hours or working groups would help them, I hope this might be a place we can answer very beginner questions or direct them towards the other available resources that are more specific to their issue.

Reply:
Ah, that might be a feature after all then. I don't insist on a change. :-)